Replace `datadog/agent` docker images by `gcr.io/datadoghq/agent` and remove `DOCKER_CONTENT_TRUST=1` (#11864)
DataDog/documentation: content/en/agent/guide/container-images-for-docker-environments.md

{{% /tab %}}
{{% tab "Docker Hub" %}}

| Datadog service                         | Docker Hub                               | Docker Pull Command                                        |
|-----------------------------------------|------------------------------------------|------------------------------------------------------------|
| [Docker Agent][1]                       | [Docker Agent (v6+)][2]                  | `DOCKER_CONTENT_TRUST=1 docker pull datadog/agent`         |
| Docker Agent (v 5)                      | [Docker Agent (v5)][3]                   | `docker pull datadog/docker-dd-agent`                      |
| [DogStatsD][4]                          | [DogStatsD][5]                           | `DOCKER_CONTENT_TRUST=1 docker pull datadog/dogstatsd`     |
| [Datadog Cluster Agent][6]              | [Cluster Agent][7]                       | `DOCKER_CONTENT_TRUST=1 docker pull datadog/cluster-agent` |
| [Synthetics Private Location Worker][8] | [Synthetics Private Location Worker][9]  | `docker pull synthetics-private-location-worker`           |

> Docker Hub supports [content trust][10] for the `agent`, `cluster-agent` and `dogstatsd` images.

Reviewer: ```suggestion
Docker Hub supports [content trust][10] for the `agent`, `cluster-agent`, and `dogstatsd` images.
```

> Enabling docker content trust for those images by setting `DOCKER_CONTENT_TRUST=1` provides extra guarantee that the images haven’t been tampered.

Reviewer: ```suggestion
To ensure that the images are not tampered with, enable content trust by setting `DOCKER_CONTENT_TRUST=1`.
```
